### PR TITLE
[docs] add the Stable diffusion with Jax/Flax Guide into the docs

### PR DESCRIPTION
**Pull request by yiyixuxu**

### CODE REVIEW

File: docs/source/en/using-diffusers/stable_diffusion_jax_how_to.mdx [hunk 1]
```diff
@@ -0,0 +1,250 @@
...
+
+Note that JAX is not exclusive to TPUs, but it shines on that hardware because each TPU server has 8 TPU accelerators working in parallel.
+
+## Setup
+
+First make sure diffusers is installed.
+
+```bash
+!pip install flax transformers ftfy
+!pip install diffusers==0.9.0
```
Comment (patrickvonplaten) [inline docs/source/en/using-diffusers/stable_diffusion_jax_how_to.mdx:19]:
```suggestion
!pip install diffusers
```

would be nice if we manage to keep things up to date

**Review by patrickvonplaten — APPROVED** [review]
Cool, @yiyixuxu could you test this once to make sure everything works with current diffusers version (0.14.0) ?

**Comment by patrickvonplaten** [comment]
And yes we should definitely add `FlaxStableDiffusionPipeline` etc... to the docs as well

**Comment by yiyixuxu** [comment]
@patrickvonplaten I think flax stopped working on colab again 😭😭😭
https://github.com/google/flax/issues/2861

**Comment by yiyixuxu** [comment]
@stevhliu 
I will add the Flax pipelines to API - do you think we should add it to the same page as its pytorch counterpart? 
e.g. we can add `FlaxStableDiffusionPipeline` to this [page](https://moon-ci-docs.huggingface.co/docs/diffusers/pr_2487/en/api/pipelines/stable_diffusion/text2img)

It seems that Flax/Jax section in README  only contains the most basic use examples - I think these examples needed to be added to docstrings so they would show up on the flax pipelines' API page. What do you think?

**Comment by krahnikblis** [comment]
> @patrickvonplaten I think flax stopped working on colab again 😭😭😭 [google/flax#2861](https://github.com/google/flax/issues/2861)

hey i've been wrestling with this specific issue (Flax HF pipelines on Colab) for weeks and finally got past the version compatibility issue, at least with Flax; check the workaround in the [issue post here](https://github.com/google/flax/issues/2950) - might be useful but i'm not sure how the automated setup of Diffusers/Transformers dependencies would handle the recursive dependency of Flax<>Orbax; i.e. if my workaround method could be used here, or required to be done before installing HF libs...

**Comment by patrickvonplaten** [comment]
Gentle ping here @yiyixuxu

**Pull request merged**